.github/workflows/run-tests.yml

```
@@ -0,0 +1,45 @@
+# This file is part of git-big-picture
+#
+# Copyright (C) 2020 Sebastian Pipping <sebastian@pipping.org>
+#
+# git-big-picture is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# git-big-piture is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with git-big-picture.  If not, see <http://www.gnu.org/licenses/>.
+
+name: Tests
+
+on:
+- pull_request
+- push
+
+jobs:
+  run-tests:
+    name: Tests
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2.3.4
+    - uses: actions/setup-python@v2.1.4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install build dependencies
+      run: |
+        sudo apt-get install --no-install-recommends -y graphviz
+        pip install -r test_requirements.txt
+    - name: Run the test suite
+      run: |
+        git config --global user.email tests@example.com
```

```suggestion
        git config --global user.email tests@git-big-picture.test
```

https://en.m.wikipedia.org/wiki/.test

---

Okay interesting, why not :smiley: Will repush in a minute.